mirascope/core/base/_call_factory.py

```diff
@@ -42,7 +47,9 @@
 _BaseCallParamsT = TypeVar("_BaseCallParamsT", bound=BaseCallParams)
 _BaseDynamicConfigT = TypeVar("_BaseDynamicConfigT", bound=BaseDynamicConfig)
 _BaseStreamT = TypeVar("_BaseStreamT", bound=BaseStream)
+_SyncBaseClientT = TypeVar("_SyncBaseClientT", bound=object)
```

Comment:
nit: move this below the `_BaseClientT` line right below so this is right above it's async counterpart